INT-3700

loesak opened the pull request:
Change to calculate proper syndicate entry date for comparison when
adding feeds for processing.

I have signed and agree to the terms of the SpringSource Individual
Contributor License Agreement.

loesak committed:
INT-3700

Change to calculate proper syndicate entry date for comparison when
adding feeds for processing.

I have signed and agree to the terms of the SpringSource Individual
Contributor License Agreement.

artembilan commented:
Great! Thank you @loesak !
But I'm forced to ask you to fully follow with Contribution Guideline: https://github.com/spring-projects/spring-integration/blob/master/CONTRIBUTING.md.
First of all you need to create a branch `INT-3700` and PR from the not from `master`.
And it would be good to have some test on the matter.
As well as follow, please, with first commit message rule.

I consider this fix as a backport for all maintenance versions.

Please, be patient with our review and it save our and your time for the future contributions!

loesak commented:
Feel like a dope right now. I dont use GIt much so I dont know how to branch. I've also never used Gradle so i'm having a hard time getting my project to compile dependencies to fix the tests.

Will work on this today...

loesak commented:
i'm not sure if your intention was for me to fix how i checked in but i created another pull request. I created a branch like stated in contributors document. I added a unit tests, and i updated my comment based on first commit rule.

https://github.com/spring-projects/spring-integration/pull/1426

artembilan commented:
Great! Yes, that's correct.
Thank you @loesak !

Will come back to you tomorrow on that new PR!
